example/issue-1377-circle-event-probs.html

```diff
@@ -0,0 +1,200 @@
+              }
+            };
+
+            $scope.spiderOptions = {
+                keepSpiderfied:true
+            };
+
+            $scope.clusterOptions = {
+                maxZoom:15,
+                styles: [{
+                  width: 50,
+                  height: 50,
+                  textColor: '#ffffff',
+                  textSize: 15,
+                  url:"http://google-maps-utility-library-v3.googlecode.com/svn/tags/markerclusterer/1.0.2/images/m1.png"
+                }],
+                keepSpiderfied:true
+            };
+
+            $scope.map = {
+                center: { latitude: 46.83245, longitude: -71.23267}, zoom: 11, bounds: {}, markers: [], dragging: false,
+                events:
+                {
+                    zoom_changed: function(map){
+                        if(map.getZoom() > 14){
+                            $scope.map.doSpider = true;
+                        }
+                        else{
+                            $scope.map.doSpider = false;
+                        }
+                    }
+                },
+                me:{
+                  id: 1,
+                  control:{},
+                  center: {
+                    latitude: 46.83245,
+                    longitude: -71.23267
+                  },
+                  radius: 10*1000,
+                  stroke: {
+                      color: '#08B21F',
+                      weight: 3,
+                      opacity: 1
+                  },
+                  fill: {
+                      //color: '#08B21F',
+                      color: '#1D333C',
+                      opacity: 0.1
+                  },
+                  geodesic: false, // optional: defaults to false
+                  draggable: false, // optional: defaults to false
+                  clickable: false, // optional: defaults to true
+                  editable: true, // optional: defaults to false
+                  visible: true, // optional: defaults to true
+                  control: {},
+                  events:{
+                    dragend:function(circle, eventName, model, args){
+                      console.log('Event - dragend', eventName)
+                      $scope.debug_message.push('event: dragend')
+                    },
+                    radius_changed:function(circle, eventName){
+                      console.log('Event - radius_changed', eventName)
+                      $scope.debug_message.push('event: radius_changed')
+                    },
+                    center_changed:function(circle, eventName){
+                      console.log('Event - center_changed', eventName)
+                      $scope.debug_message.push('event: center_changed')
+                    }
+                  }
+                }
+            };
+
+            uiGmapIsReady.promise().then(function (map_instances) {
+                $scope.map.type = 'cluster';
+                $scope.infoBox = new InfoBox({
```

Comment:
@sam2x notice you dont need the google maps script and infobox script loaded in html if you wait for google to be defined here. InfoBox is wrapped into the directive suite and google maps is loaded for you.